utils/build-script-impl

```diff
@@ -2361,6 +2360,13 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
+                if [ "${SWIFT_INSTALL_COMPONENTS}" ] ; then
```

Comment:
I think generally we have been using [[ and -n. Can you change to use that?

Reply:
I agree w.r.t. `[[`, but we aren't really using `-n`.  I did a quick and dirty search of build-script-impl and I see:

* `[[ $x ]]` x 66
* `[[ -n $x ]]` x 4
* `[[ $x != ""]` x 4

Reply:
Changed to `if [[ "${SWIFT_INSTALL_COMPONENTS}" ]] ; then`.